detectorSegmentations/include/detectorSegmentations/FCCSWGridModuleThetaMergedHandle_k4geo.h

```diff
@@ -0,0 +1,124 @@
+  template <typename Q>
+  FCCSWGridModuleThetaMerged_k4geo(const Handle<Q>& e) : Handle<Object>(e) {}
+  /// Assignment operator
+  FCCSWGridModuleThetaMerged_k4geo& operator=(const FCCSWGridModuleThetaMerged_k4geo& seg) = default;
+  /// Equality operator
+  bool operator==(const FCCSWGridModuleThetaMerged_k4geo& seg) const { return m_element == seg.m_element; }
+
+  /// determine the position based on the cell ID
+  inline Position position(const CellID& id) const { return Position(access()->implementation->position(id)); }
+
+  /// determine the cell ID based on the position
+  inline dd4hep::CellID cellID(const Position& local, const Position& global, const VolumeID& volID) const {
+    return access()->implementation->cellID(local, global, volID);
+  }
+
+  /// access the grid size in theta
+  inline double gridSizeTheta() const { return access()->implementation->gridSizeTheta(); }
+
+  /// access the coordinate offset in theta
+  inline double offsetTheta() const { return access()->implementation->offsetTheta(); }
+
+  /// access the number of theta cells merged for given layer
+  inline int mergedThetaCells(const int layer) const { return access()->implementation->mergedThetaCells(layer); }
+
+  /// access the number of modules
+  inline int nModules() const { return access()->implementation->nModules(); }
+
+  /// access the number of layers
+  inline int nLayers() const { return access()->implementation->nLayers(); }
+
+  /// access the number of merged modules for given layer
+  inline int mergedModules(const int layer) const { return access()->implementation->mergedModules(layer); }
+
+  /// set the coordinate offset in theta
+  inline void setOffsetTheta(double offset) const { access()->implementation->setOffsetTheta(offset); }
+
+  /// set the grid size in theta
+  inline void setGridSizeTheta(double cellSize) const { access()->implementation->setGridSizeTheta(cellSize); }
+
+  /// access the field name used for theta
+  inline const std::string& fieldNameTheta() const { return access()->implementation->fieldNameTheta(); }
+
+  /// access the field name used for module
+  inline const std::string& fieldNameModule() const { return access()->implementation->fieldNameModule(); }
+
+  /// access the field name used for layer
+  inline const std::string& fieldNameLayer() const { return access()->implementation->fieldNameLayer(); }
+
+
+  /** \brief Returns a std::vector<double> of the cellDimensions of the given cell ID
+      in natural order of dimensions (nModules, dTheta)
+
+      Returns a std::vector of the cellDimensions of the given cell ID
+      \param cellID
+      \return std::vector<double> size 2:
+      -# size in module
+      -# size in theta
+  */
+  inline std::vector<double> cellDimensions(const CellID& /*id*/) const {
+    //return {access()->implementation->gridSizePhi(), access()->implementation->gridSizeTheta()};
+    // not implemented
+    return {0., 0.};
```

Comment:
Can this be implemented? In Particular the comment promises an implementation that is not delivered.

Can it otherwise throw an exception so that we spot this clearly?

Reply:
Please add a phrase in the docstring that this is not implemented.